sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -335,7 +335,7 @@ try
 {
     // Receive messages using the receiver client
 }
-catch (ServiceBusExceptions ex) where 
+catch (ServiceBusExceptions ex) when
```

Comment:
Really we should use a code snippet here so this type of error would not be possible.

Reply:
Agreed.  They didn't exist back when the original ReadMe was written; it would be great to have a readme test to back these up with snippets.

Reply:
This code has no executing statements (only that catch expression exists). So, I don't think it warrants a snippet.

Reply:
You can put the code in an existing sample test and just wrap it in a new snippet region. This way the code will be compiled before it gets inserted into the ReadMe.

Reply:
@jsquire Also, it says `ServiceBusExceptions` (with an 's'). If you have that in EventHubs (but for EventHubException), you should fix it.

Reply:
@JoshLove-msft Added a snippet for this. 😁